### PR TITLE
Prepare Release v1.11.0

### PR DESCRIPTION
**Pull request by giladgray**
:fireworks: **Highlights**: two new components `DateRangeInput` and `Tabs2` (rewrite of `Tabs`), some really great bugfixes.

:book: **Latest docs**: [blueprintjs.com/docs](http://blueprintjs.com/docs)

## General
- 📦  __Fixed__ added `browser` and `webpack` entries to each package.json so unpkg will serve the `.bundle.js` file #621
- 📦  __Fixed__  `.bundle.js` files now export to the global `Blueprint` variable, namespaced by package: `Blueprint.Core.Button`, `Blueprint.Table.Table` etc. #631

## @blueprintjs/core `1.11.0`
- 🌟  **NEW** `Tabs2` is a rewrite of `Tabs` with a simpler and more flexible API #261 #722  
  ```jsx
  <Tabs2 id="routes" onChange={this.handleTabChange} selectedTabId="pulls">
      <Tab2 id="code" title="Code" panel={<Code />} />
      <Tab2 id="issues" title="Issues" panel={<Issues />} />
      <Tab2 id="pulls" title="Pull Requests" panel={<PullRequests />} />
      <Tabs2.Expander />
      <input className="pt-input" type="text" placeholder="Search..." />
  </Tabs2>
  ```
  - This new API is offered in a backwards-compatible manner by appending a `2`. The original `Tabs` components are still available under their old names but are _deprecated and will be replaced in v2.0._
  - Only two components (`Tabs2` and `Tab2`) are needed, rather than the previous four.
  - Selection is managed by ID, rather than by index. This is more reliable and deterministic and does not require translating between numbers and tab names. It does, however, require that every `Tab2` have a locally unique `id` prop (which also resolves SSR issues #384).
  - Arbitrary elements are supported in the tab list, and order is respected. Yes, you can even insert things _between_ tabs.
  - CSS API remains exactly the same.
- :star2: __Fixed__ insidious cursor/selection rendering bug with multiple `Popover`s in Firefox #406
- __Fixed__ `NumericInput` & `Button` React prop warnings #736 #753
- __Fixed__ `Tree` error when removing nodes #741
- __Improved__ dark link color for more contrast #764
- 📠  __Deprecated__ `Tabs` component in favor of `Tabs2` #767 
  - a console warning will appear during development (not in production)

## @blueprintjs/datetime `1.9.0`
- 🌟  **NEW** DateRangeInput component
release-daterangeinput
  - A `.pt-control-group` of two `InputGroup`s that shows a `DateRangePicker` on focus.
  - Supports selecting dates by typing in the input fields, clicking dates in the calendar, or clicking shortcuts in the popover.
  - Previews the date range in the calendar and in the input fields as you move your cursor within the calendar.
  - Supports controlled and uncontrolled usage.
- __Fixed__ `DatePicker` selected day is always blue #740
- __Fixed__ `DatePicker` outside days appearance #740

## @blueprintjs/table `1.8.0`
- 🌟 __Fixed__ `JSONFormat` `detectTruncation` properly detects truncation #760
image
- __Fixed__ `Column` component respects `className` prop #690 
- __Fixed__ `TruncatFormated` `Popover` scrollbars only appear when needed #720 
- __Fixed__ react-hot-loader support by `console.warn`ing instead of throwing errors #723

## Documentation
- __Added__ `active` switch to `Button` example (:tophat: @DrewDennison) #749

### CODE REVIEW

File: docs/docs/docs.css [hunk 1]
```diff
@@ -6574,17 +6577,35 @@ table.pt-table.pt-interactive tbody tr:hover td {
...
+    border-bottom-right-radius: 0; }
+.pt-daterangepicker .DayPicker-Day--hovered-range-end:not(.DayPicker-Day--hovered-range-start) {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0; }
+.pt-dark .pt-daterangepicker .DayPicker-Day--hovered-range:not(.DayPicker-Day--selected):not(.DayPicker-Day--selected-range):not(.DayPicker-Day--selected-range-start):not(.DayPicker-Day--selected-range-end) {
+    background-color: #394b59; }
+.pt-dark .pt-daterangepicker .DayPicker-Day--selected-range {
+    background-color: #394b59; }
+.pt-dark .pt-daterangepicker .DayPicker-Day--selected-range:hover {
+      background-color: #5c7080; }
```
Comment (giladgray) [inline docs/docs/docs.css]: these css diffs are so cool 🕶

**Comment by adidahiya** [comment]
cell truncation looks better 👍 

image

were you trying to vertically center this search input in the tab list?

image

**Comment by cmslewis** [comment]
@giladgray are we going to fill out the Highlights?

**Comment by giladgray** [comment]
🔢 fixing search input, filling out highlights, releasing blueprint ✈️

**Pull request merged**